eng/pipelines/pipeline-generation.yml

```diff
@@ -4,7 +4,7 @@ trigger: none
 
 variables:
   skipComponentGovernanceDetection: true
-  PipelineGeneratorToolVersion: 1.0.2-dev.20200423.1
+  PipelineGeneratorToolVersion: 1.0.2-dev.20200502.1
```

Comment:
@weshaggard Crazy question but where is this version coming from?

Reply:
That is the default build numbers coming from our tools build pipeline. Similar to our daily dev versions numbers.